content/en/blog/plugins-early-access.md

```diff
@@ -0,0 +1,147 @@
+
+#### Running Plugins in Falco
+
+For now, when plugin support is enabled, Falco's core support for syscalls is disabled. Additionally, only a single source plugin can be loaded at once. If multiple source plugins are loaded, falco will return an error. Any number of extractor plugins can be loaded at once.
+
+We do plan on relaxing these restrictions to allow running multiple source plugins at once, once we tackle the concurrency, fairness, and resource utilization issues that result from having multiple streams of events being processed by a single falco rules engine.
+
+### Cloudtrail Plugin
+
+Along with the introduction of plugins support, we've written a source plugin [cloudtrail](https://github.com/falcosecurity/plugins/tree/master/plugins/cloudtrail) that can read cloudtrail logs and pass them to the rules engine as events.
+
+The plugin can be configured to obtain log files in a number of ways, depending on the `open_params` value in `falco.yaml`:
+
+* A [S3](https://aws.amazon.com/s3/) bucket: `s3://<S3 Bucket Name>[/<Optional Prefix>]`.
+* A [SQS](https://aws.amazon.com/sqs/) queue that passes along [SNS notifications](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/configure-sns-notifications-for-cloudtrail.html) about new log files: `sqs://<SQS Queue Name>`
+* A local filesystem path: `Any path`.
+
+The plugin also exports fields that extract information from a cloudtrail event, such as the event time, the aws region, S3 bucket/EC2 instance name, etc.
+
+The plugin uses the same authentication mechanisms used by the [AWS Go SDK](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials):
+
+* Environment Variables: specify the aws region with AWS_REGION=xxx, the access key id with AWS_ACCESS_KEY_ID=xxx, and the secret key with AWS_SECRET_ACCESS_KEY=xxx.
+* Shared Configuration Files: specify the aws region in a file at $HOME/.aws/config and the credentials in a file at $HOME/.aws/credentials.
+
+#### New Rules for AWS Cloudtrail
+
+Along with the plugin, we've created new Falco rules that look for suspicious/notable/interesting activity in cloudtrail logs. Here is an example rule:
+
+```yaml
+- rule: Console Login Without MFA
+  desc: Detect a console login without MFA.
+  condition:
+    ct.name="ConsoleLogin" and ct.error=""
+    and json.value[/userIdentity/type]!="AssumedRole" and json.value[/responseElements/ConsoleLogin]="Success"
+    and json.value[/additionalEventData/MFAUsed]="No"
+  output:
+    Detected a console login without MFA (requesting user=%ct.user, requesting IP=%ct.srcip, AWS region=%ct.region)
+  priority: CRITICAL
+  source: aws_cloudtrail
+```
+
+As the source is `aws_cloudtrail`, it is run against any events returned by the cloudtrail plugin. Also note that the rule condition uses a mix of fields from the cloudtrail plugin (`ct.name`, `ct.error`) as well as fields from the json plugin (`json.value` with json_pointer argument).
+
```

Comment:
I know Falco people already knows how events look. But new cloud people might be arriving at this article and they might need more background info.

It may be nice to add how the output of the rule looks like. i.e.:

If the previous rule were to be triggered, you would get the following alert:
The alert output.

You can learn more on how to handle this alerts on this doc page, and the falco sidekick project.

Reply:
Sure, added a sample alert and linked both to alerts page and falcosidekick blog post (I don't think there's a docs page for it).